src/lib/bootloader/stage1_device.rb

```diff
@@ -102,18 +100,17 @@ def underlaying_devices_one_level(dev)
     #
     # Ignores physical volumes on a whole disk. See bnc#980529
     #
-    # @param volumes_group_list [Y2Storage::LvmVgsList]
+    # @param volume_groups_list [Y2Storage::LvmVgsList]
     # @return [Y2Storage::LvmPvsList]
-    def usable_pvs(volume_groups_list)
-      pvs = volume_groups_list.physical_volumes
-      pvs.with { |pv| !Storage.disk?(pv.blk_device) }
+    def usable_pvs(volume_group)
+      volume_group.lvm_pvs.reject { |pv| pv.blk_device.is?(:disk) }
```

Comment:
Better with `pv.plain_blk_device.is?(:disk)`. Otherwise, encrypted PVs on top of a whole disk would not be discarded.